test/scripts/traffic-flow-tests.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
```

Comment:
Just curious, was there a reason why bash was chosen?

Reply:
That is what all the other scripts in this folder use; did not want to be too different.
as you can see this script is not meant to be doing anything fancy. We save that for the tests themselves. ;)